torch_geometric/data/meshcnnhandler.py

```diff
@@ -0,0 +1,317 @@
+from tempfile import mkstemp
+from shutil import move
+import torch
+import numpy as np
+import os
+from ..nn.mesh.mesh_union import MeshUnion
+from torch_geometric.data import Data
+
+
+class MeshCNNData(Data):
```

Comment:
Is this class needed. Could the `Data` class be used instead.
`MeshCnnData = Data(self.pos = pos,self.edges = edges,self.sides = sides,self.edges_count = edges_count,self.ve = ve...`. What do you think.